### ISSUE TITLE
Upcoming match-v4 deprecation

### ISSUE THREAD

**Issue by bcho04** [bcho04/galeforce#8]
All match-v4 endpoints will be deprecated on **August 23, 2021** in compliance with Riot's 60-day deprecation timer and the upcoming release of the match-v5 endpoints. `galeforce.lol.match` will need to be rewritten *in a manner that limits breaking changes* and associated documentation within the project's README and docblocks will need to be updated accordingly.

**Comment by bcho04** [comment]
Resolved after #11 was merged into the `develop` branch.
